packages/suite/src/components/suite/NavigationBar/components/NavigationActions/index.tsx

```diff
@@ -132,6 +136,15 @@ const NavigationActions = (props: Props) => {
                 );
             })}
             <ActionsContainer desktop={isDesktop()} mobileLayout={props.isMobileLayout}>
+                <ActionItem
+                    onClick={() => {
+                        setTheme(theme.variant === 'light' ? 'dark' : 'light');
+                    }}
+                    isActive={theme.variant === 'dark'}
+                    label={<Translation id="TR_DISCREET" />}
```

Comment:
This is one of the todos you mentioned for a future PR, right?

Reply:
this will get fixed (removed) in this PR :) We are going to need only switch in the settings.